extensions/ipynb/package.json

```diff
@@ -1,133 +1,138 @@
 {
-	"name": "ipynb",
```

Comment:
the formatter auto-switched this from tabs to spaces, which matches other package.json files

Reply:
actually, it's not consistent, but matches with the root level file, and my default formatter settings.